report.php

```diff
@@ -49,6 +49,12 @@
 echo $output->header();
 // Teacher can see results.
 if (has_capability('mod/sortvoting:readresponses', $modulecontext)) {
+    // Check to see if groups are being used here.
```

Comment:
You shouldn't need to "check" anything, just call your method unconditionally on line 55

`$currentgroup` is unused
Also pass `\moodle_url` instance as second parameter